tests/unit/src/main/scala/tests/BaseWorksheetLspSuite.scala

```diff
@@ -1,15 +1,15 @@
 package tests
 
-import scala.meta.internal.metals.ClientExperimentalCapabilities
 import scala.meta.internal.metals.UserConfiguration
-import scala.meta.internal.metals.MetalsSlowTaskResult
 import scala.concurrent.Promise
+import scala.meta.internal.metals.MetalsSlowTaskResult
+import scala.meta.internal.metals.ClientExperimentalCapabilities
 
 abstract class BaseWorksheetLspSuite(scalaVersion: String)
     extends BaseLspSuite("worksheet") {
   override def experimentalCapabilities
       : Option[ClientExperimentalCapabilities] =
-    Some(ClientExperimentalCapabilities(decorationProvider = true))
+    Some(ClientExperimentalCapabilities.Default.copy(decorationProvider = true))
```

Comment:
`new ClientExperimentalCapabilities(decorationProvider = true) ` ?

Reply:
Yea, this won't work anymore because of the type of constructor we have.

Reply:
Ach, yeah. You are right!